dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -239,10 +287,40 @@ def install(self):
             print("Build DALI TF library:\n\n " + cmd + '\n\n')
             subprocess.check_call(cmd, cwd=self.src_path, shell=True)
 
-            if not self.check_import(lib_path, dali_stub_lib):
-                raise ImportError("Error while importing the DALI TF plugin built from source, will not install")
+            if not self.check_plugin(lib_path, dali_stub_lib):
+                raise ImportError("Error while loading or testing the DALI TF plugin built from source, will not install")
             print("Installation successful")
 
+    def check_install_env(self):
+        print("Checking build environment for DALI TF plugin ...")
+        print(self.debug_str())
+
+        Path(self.plugin_dest_dir).mkdir(parents=True, exist_ok=True)
```

Comment:
nitpick: Maybe it's better to first make a check that may raise an error and only if it passes, create the directory?